### PR TITLE
Filings UI - ToDo list updates

### PR DESCRIPTION
**Pull request by cameron-eyds**
*Issue #:* /bcgov/entity#6923

*Description of changes:*

* Updates local config routing

* Updates ToDo list verbiage surrounding Alterations ( Initial required alteration and subsequent company info changes)

* Applies UX updates to Not in GoodStanding condition

* Moved some styling to global.

* Updated App and SBC dependency version.

* Minor test updates.

*Testing Notes:*

* BC1230105 - Limited Company NOT IN GOOD STANDING with a REQUIRED Alteration

* BC1230111 - Limited Company IN GOOD STANDING with a REQUIRED Alteration BC1230001

* BC1230001 - Benefit Company IN GOOD STANDING with a DRAFT Alteration

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

File: src/assets/styles/base.scss [hunk 1]
```diff
@@ -72,6 +72,14 @@ p {
   color: $app-blue !important;
 }
 
+.bold-text {
```
Comment (severinbeauvais) [inline src/assets/styles/base.scss:75]: Why not use `font-weight-bold`?

Comment (cameron-freshworks) [reply src/assets/styles/base.scss:75]: Updated 👍

File: src/components/Dashboard/TodoList.vue [hunk 2]
```diff
@@ -1279,12 +1326,16 @@ export default {
...
+    p {
+      color: $gray7;
+    }
   }
 }
 
 .todo-item .list-item {
   padding: 0;
   justify-content: space-evenly;
+  align-items: flex-start;
```
Comment (severinbeauvais) [inline src/components/Dashboard/TodoList.vue:1338]: I think there's a Vuetify class for this. Not sure if that helps here.

**Review by severinbeauvais — APPROVED** [review]
Nothing major. Fix if it makes sense.

File: src/components/Dashboard/TodoList.vue (hunk 2 above)
Comment (cameron-freshworks) [reply src/components/Dashboard/TodoList.vue:1338]: Definitely helps to know but in this case we would be putting some styling here and some in the template. So i opted to keep together i think 👍

**Review by vysakh-menon-aot — APPROVED** [review]

**Pull request merged**